docs/api/v1/resourcesetinputprovider.md

```diff
@@ -285,6 +285,70 @@ spec:
 ```
 
+### Schedule
+
+The `.spec.schedule` field is optional and can be used to specify a list of `Schedule` objects.
+
+Each `Schedule` object has the following fields:
+- `.cron`: a required string representing the cron schedule in the format accepted by
+  [cron](https://crontab.guru/).
+- `.timeZone`: a string representing the time zone in which the cron schedule should be interpreted.
+  This is optional and defaults to `UTC`.
+- `.window`: an optional string representing the time window duration in which reconciliations are
+  allowed to run. The format is a Go duration string, such as `1h30m` or `2h45m`. Defaults to `0s`,
+  meaning no window is applied.
+
+Example:
+
+```yaml
+spec:
+  schedule:
+  - cron: 0 8 * * 1-5 # Every weekday at 8:00 AM
+    timeZone: Europe/London
+    window: 8h
+```
+
+When multiple schedules are specified, flux-operator will:
+- Reconcile the ResourceSetInputProvider if at least one of the schedules matches the current time.
+- Use the earliest next scheduled time across all schedules to determine the next scheduled time.
+
+When a schedule is specified with a zero duration window (the default), flux-operator will make
+the best effort to reconcile the ResourceSetInputProvider at the scheduled time, but it may not be
+able to guarantee that the reconciliation will start exactly at that time, especially if the operator
+is too busy or if the cluster is under heavy load. If multiple schedules are specified and at least
+one has a zero duration window, flux-operator will always reconcile the ResourceSetInputProvider upon
+any requests.
+
+When a schedule is specified with a non-zero duration window, flux-operator will only reconcile
+the ResourceSetInputProvider when the time point `time.Now().Add(obj.GetTimeout())` falls within
+the time window defined by the schedule. This check is performed not only at the start of the
+reconciliation, but also when scheduling the next reconciliation according to the interval defined
+by the `fluxcd.controlplane.io/reconcileEvery` [annotation](#reconciliation-configuration).
+
+To force a one-off out-of-schedule reconciliation, annotate the ResourceSetInputProvider with
+`reconcile.fluxcd.io/requestedAt: "<current timestamp>"` and
+`reconcile.fluxcd.io/forceAt: "<current timestamp>"`. Both annotations must be annotated
+with the exact same string, otherwise the reconciliation will not be triggered. This can
+be done easily using the `flux-operator` CLI:
+
+```shell
+flux-operator reconcile inputprovider <name> --force
+```
+
+When the next reconciliation of the ResourceSetInputProvider is due to a schedule
+(and not due to the interval defined by the `fluxcd.controlplane.io/reconcileEvery`
+annotation), the status field `.status.nextSchedule` holds information about the next
+scheduled reconciliation. For example:
+
+```yaml
+status:
+  nextSchedule:
+    cron: 0 8 * * 1-5
+    timeZone: Europe/London
+    when: "2025-06-29T00:00:00Z"
+    window: 8h
+```
```

Comment:
This should be moved to the Status section.

The explanation can be simplified:

When `.spec.schedule` is set, the next scheduled reconciliation is reported in `.status. nextSchedule`.

Reply:
That's not true, though. During the window we don't report anything, the `.status.nextSchedule` field is empty. The interval is what is scheduling the next reconciliation, so there's no point in reporting this in the status during the window.

I moved this section to Status and modified the explanation so it starts simpler, and then we have a paragraph at the end explaining what happens during a window. Thanks!